inference-engine/tests/functional/inference_engine/ngraph_reader/convolution_backprop_data_tests.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadConvolutionBackpropDataNetwork) {
+    std::string model = R"V0G0N(
+<net name="unet-camvid-fp32-onnx-0001" version="10">
```

Comment:
I think will be better to change the name. If I understand right the hardcoded IR is not equal to unet-camvid-fp32-onnx-0001 😄